src/main/cpp/vision/DragonLimelight.cpp

```diff
@@ -113,6 +136,12 @@ bool DragonLimelight::IsLimelightRunning()
     return false;
 }
 
+/// ----------------------------------------------------------------------------------
+/// @brief Retrieve detected AprilTag targets from Limelight.
+/// @param validAprilTagIDs Optional list of field april tag IDs to filter results. If empty, all tags returned.
+/// @return vector of DragonVisionStruct unique_ptr for each valid AprilTag detection. Empty if NT not available.
+/// @notes Each DragonVisionStruct contains offsets, area, latency and distances computed by Limelight helpers.
```

Comment:
/// @notes Each DragonVisionStruct contains offsets, area, latency and distances computed by Limelight helpers.

I think this is okay, but we do need to make sure we actually are calculating the distance yet. I also thought we were going to use the pose util to get the distance. I just want to make sure we document correctly if we continue out plan